lib/tasks/gitlab/check.rake

```diff
@@ -278,6 +279,29 @@ namespace :gitlab do
       end
     end
 
+    def check_upload_perms
+      print "Upload directory permissions correct? ... "
+
+      upload_path = Rails.root.join(Gitlab.config.gitlab.user_home,"gitlab-rails","uploads")
+      upload_path_tmp = Rails.root.join(upload_path,"tmp")
```

Comment:
I think a better way to find the uploads path might be to take `File.realpath(Rails.root.join('public/uploads'))`.